### ISSUE TITLE
Downranked Rejuv/Regrowth HOTS not shown

### ISSUE THREAD

**Issue by KineticConundrum** [Aviana/HealComm#27]
Downranked Healing Touch shows, and downranked Regrowth initial heal do show, but the HOTS do not appear.

**Comment by Aviana** [comment]
Fixed in latest release-

**Comment by KineticConundrum** [comment]
Still not working for me. Downranked Rejuv/Regrowth hots not showing.

**Comment by Aviana** [comment]
I tested this with a lvl 10 druid and using rank 1 and 2 of reju on himself. Can you at least confirm that that is working for you?

**Comment by KineticConundrum** [comment]
Rank 1 and 2 are not working, but I don't have a level 10 druid to test with specifically.

**Comment by DevLazio** [comment]
I just tested release 2.04 (Lib version)  with a lvl 60 druid and it doesn't work, at all.
Now, even max ranks HOTS are not showing up.

Using a previous version (the one included with Elvui) I think I found the issue with HOTS downranking.  That's because you broadcast the max lvl ID on line 1593 (sendMessage(string.format("H::%d:%d:%d:%s", spellID, amount, duration, targets))) and not the downranked id.
Hope that helps. I will try to get fix it and send a pull request if I have some time available.
